test/correctness/div_mod.cpp

```diff
@@ -50,7 +50,7 @@ uint64_t ubits(int unique, int i, int j) {
     bits = (bits ^ (bits >> 32)) * mu;
     return bits;
 }
-
+Target target;
```

Comment:
In fairness, I think a lot of the other tests do this. But, I agree, we should not be introducing new instances.